Windows build support

https://ci.appveyor.com/project/r9y9/hts-engine-api/build/1.0.4/job/f5tsfq8qu1d2ibvu

```
C:\MinGW\msys\1.0\bin\sh --login /c/projects/hts-engine-api/msys_build.sh
Downloading mingw-w32-bin-i686-20140102.7z
Setting top to                           : c:\projects\hts-engine-api\src 
Setting out to                           : c:\projects\hts-engine-api\src\build 
Checking for waf version in 1.7.11-1.9.0 : ok 
Checking for 'gcc' (C compiler)          : C:\MinGW\msys\1.0\home\appveyor\mingw32\bin\gcc.exe 
Checking for header stdlib.h             : yes 
Checking for header string.h             : yes 
 
hts_engine_API has been configured as follows:
 
[Build information]
Package:                 hts_engine_API-1.0.9
build (compile on):      x86-win32
host endian:             little
Compiler:                gcc
Compiler version:        4.8.2
CFLAGS:                  -O2 -Wall -fno-common -Wstrict-prototypes
 
'configure' finished successfully (0.955s)
Waf: Entering directory `c:\projects\hts-engine-api\src\build'
[ 1/13] Compiling lib\HTS_audio.c
[ 2/13] Compiling lib\HTS_engine.c
[ 3/13] Compiling lib\HTS_gstream.c
[ 4/13] Compiling lib\HTS_label.c
[ 5/13] Compiling lib\HTS_misc.c
[ 6/13] Compiling lib\HTS_model.c
[ 7/13] Compiling lib\HTS_pstream.c
[ 8/13] Compiling lib\HTS_sstream.c
[ 9/13] Compiling lib\HTS_vocoder.c
[10/13] Compiling bin\hts_engine.c
[11/13] Compiling hts_engine_API.pc.in
[12/13] Linking build\lib\hts_engine_API-1.dll
lib\HTS_audio.c.1.o:HTS_audio.c:(.text+0xbc): undefined reference to `_imp__waveOutWrite@12'
 
lib\HTS_audio.c.1.o:HTS_audio.c:(.text+0x196): undefined reference to `_imp__waveOutReset@4'
 
lib\HTS_audio.c.1.o:HTS_audio.c:(.text+0x1aa): undefined reference to `_imp__waveOutUnprepareHeader@12'
 
lib\HTS_audio.c.1.o:HTS_audio.c:(.text+0x1f4): undefined reference to `_imp__waveOutClose@4'
 
lib\HTS_audio.c.1.o:HTS_audio.c:(.text+0x3a8): undefined reference to `_imp__waveOutOpen@24'
 
lib\HTS_audio.c.1.o:HTS_audio.c:(.text+0x409): undefined reference to `_imp__waveOutPrepareHeader@12'
 
collect2.exe: error: ld returned 1 exit status
```

--- a/README.md
+++ b/README.md
@@ -22,7 +22,7 @@ git cvsimport -v \
 
 - Linux
 - Mac OS X
-- Windows (on-going)
+- Windows (gcc)
 
 ## Changes
 

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -7,11 +7,12 @@ environment:
     - COMPILER: gcc
       ARCH: "x86_64"
 
-    - COMPILER: msvc
-      ARCH: "i686"
+# Disable msvc for now
+#    - COMPILER: msvc
+#      ARCH: "i686"
 
-    - COMPILER: msvc
-      ARCH: "x86_64"
+#    - COMPILER: msvc
+#      ARCH: "x86_64"
 
 init:
   - "ECHO %PYTHON%"

--- a/src/lib/wscript
+++ b/src/lib/wscript
@@ -1,3 +1,6 @@
+import sys
+
+
 def configure(conf):
     pass
 
@@ -6,8 +9,11 @@ def build(bld):
     bld.install_files('${HPREFIX}', bld.path.ant_glob('*.h'))
     bld.install_files('${HPREFIX}', '../include/HTS_engine.h')
 
+    lib = ['winmm'] if sys.platform == "win32" else []
+
     bld.shlib(
         source=bld.path.ant_glob('*.c'),
         target=bld.env['APPNAME'],
         vnum=bld.env['VERSION'],
+        lib=lib,
         includes=['.', '../include'])

